libcloud/compute/drivers/auroracompute.py

```diff
@@ -17,13 +17,41 @@
 from libcloud.compute.drivers.cloudstack import CloudStackNodeDriver
 
 __all__ = [
+    'AuroraComputeRegion',
     'AuroraComputeNodeDriver'
 ]
 
 
+class AuroraComputeRegion(object):
+    AMS = 'Amsterdam'
```

Comment:
Minor thing - for consistency with other drivers it might be better to just use short region names (ams, rtd, mia, etc.)

That's what other drivers which support multiple regions do right now :)